ci/tox/.env-sample

```diff
@@ -3,6 +3,7 @@ GIT_VERSION=2.25.0  # note that generally the older the git version, the faster
 PYTHON_VERSION=3.8
 
 # Vars for `docker-compose run`
-BUILD_DOCS=false
+BUILD_SPHINX_DOCS=false
+CHECK_CMD_DOCS_UP_TO_DATE=false
```

Comment:
I'd rather call them `PY_DOCS` or `PYTHON_DOCS`, `CMD` is unclear in this context